tpu/src/test/java/tpu/QueuedResourceIT.java

```diff
@@ -136,4 +136,30 @@ public void testCreateQueuedResourceWithStartupScript() throws Exception {
       assertEquals(returnedQueuedResource, mockQueuedResource);
     }
   }
+
+  @Test
+  public void testCreateSpotQueuedResource() throws Exception {
+    try (MockedStatic<TpuClient> mockedTpuClient = mockStatic(TpuClient.class)) {
```

Comment:
nit: consider giving it more verbose names. It is easy to confuse between `mockedTpuClient` and `mockTpuClient`.

For example, the latter can be called `mockedClientInstance`

Reply:
Thank you! Changed name